### PR TITLE
webpack-and-babel - Jeremy Nguyen

### PR DESCRIPTION
**Pull request by jeremyhnguyen**

### CODE REVIEW

**Comment by jeremyhnguyen** [comment]
Screenshot 2023-10-11 at 10 26 44 AM

File: webpack-and-babel/imports.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo';
```
Comment (shawnkost) [inline webpack-and-babel/imports.js]: This still needs `.json`. Webpack will try and add file extensions automatically for `.js`, `.jsx`, `.ts`, `.tsx`, but not for `.json`. So we still need to explicitly state that we want a `.json` file

File: webpack-and-babel/imports.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// import from a file named `bar.js` located in the parent directory:
+import f3 from '../bar';
+
+// import from a file named `index.js` located in a directory named `baz` that
+// is in the current directory:
+import f4 from './baz/index';
+
+// import from a file named `index.js` located in a directory named `baz` that
+// is in the parent directory:
+import f5 from '../baz/index';
```
Comment (shawnkost) [inline webpack-and-babel/imports.js]: You can drop the `/index`. Webpack will automatically look for an `index.js` file for us if it wasn't able to find it before

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Jeremy!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**